assets/styles/parts/utils/_colors.scss

```diff
@@ -6,4 +6,8 @@ All variables are available here:
 https://github.com/infinum/eightshift-frontend-libs/blob/develop/styles/scss/_colors.scss
 */
 
-$base-primary-color: #FF86D6;
\ No newline at end of file
+$base-primary-color: #FF86D6;
+
+$base-text-muted-color: #999999;
+$base-aluminum-color: #EAEAEA;
+$base-white-color: #FFFFFF;
```

Comment:
all colors must go to the global colors manifest. If we define some in manifest and some in code it will become a mess very fast.